### PR TITLE
pytouhou: fix Python 3 dependency

### PR DESCRIPTION
**Pull request by queengooborg**
A scrict audit on this passed, allowing this formula to be checked off of #13133's list!

### CODE REVIEW

File: Formula/pytouhou.rb [hunk 1]
```diff
@@ -25,7 +25,7 @@ class Pytouhou < Formula
   depends_on "gtk+3" => :recommended
   if build.with? "gtk+3"
     depends_on "py3cairo" # FIXME: didn't get picked up by pygobject3 below
-    depends_on "pygobject3" => "with-python3"
+    depends_on "pygobject3"
```
Comment (vinyldarkscratch) [inline Formula/pytouhou.rb]: Realizing that this won't work as I expect it to -- misread the documentation.  Will continue my journey on this tomorrow, when I'm more awake!

**Comment by vinyldarkscratch** [comment]
Doing some tests on my own computer: had a C-level crash when keeping `depends_on "pygobject3" => "with-python3"`, but I could get `pytouhou` to launch with this new commit.

**Comment by vinyldarkscratch** [comment]
Definitely stale, but mostly because I'm waiting for a maintainer response.  😛

File: Formula/pytouhou.rb [hunk 2]
```diff
@@ -15,7 +15,7 @@ class Pytouhou < Formula
 
   option "with-demo", "Install demo version of Touhou 6"
 
-  depends_on :python3
+  depends_on "python3"
```
Comment (fxcoudert) [inline Formula/pytouhou.rb]: That's already in master

File: Formula/pytouhou.rb [hunk 3]
```diff
@@ -25,7 +25,7 @@ class Pytouhou < Formula
   depends_on "gtk+3" => :recommended
   if build.with? "gtk+3"
     depends_on "py3cairo" # FIXME: didn't get picked up by pygobject3 below
-    depends_on "pygobject3" => "with-python3"
```
Comment (fxcoudert) [inline Formula/pytouhou.rb]: It makes us pass the audit, but it will surely disable functionality. We need to think more about what to do here

**Comment by ilovezfs** [comment]
Sorry passing on this @vinyldarkscratch!

**Pull request closed without merging**